notebooks/5_local_search/index.md

```diff
@@ -0,0 +1,469 @@
+# Local Search
```

Comment:
Well done with your Lecture Note!

I've mentioned a few comments on different parts of the LN. Consider them and apply needed changes to your whole article. 
To summarize, + points about your LN are:
- Sufficient coverage of the topic
- Good delivery of concepts
- Enough Images and codes
On the other hand, - points are:
- Errors in English writing (like word misuse, punctuation, sentence construction)
- Adjustment of images and formulas
- Quality and organization of pictures (including captions and, in some cases cropping, might help)